library/common/config/config.cc

```diff
@@ -183,6 +183,7 @@ const char* config_template = R"(
               dns_failure_refresh_rate:
                 base_interval: *dns_fail_base_interval
                 max_interval: *dns_fail_max_interval
+              dns_query_timeout: 25s
```

Comment:
hard coding this in this current PR so no one is burnt by the default. I am doing a PR to make it configurable.